components/calendar/src/astronomy.rs

```diff
@@ -967,8 +967,8 @@ impl Astronomical {
     /// Based on functions from _Calendrical Calculations_ by Reingold & Dershowitz,
```

Comment:
suggestion: clarify in docs that this function takes the result of `julian_centuries(moment)` as well to avoid recalculation for optimization

Reply:
done